graph/is_graph_bipartite.cpp

```diff
@@ -0,0 +1,142 @@
+/**
+ *  @file is_graph_bipartite
+ *
+ *	@brief Algorithm to check whether a graph is bipartite
+ *	(https://en.wikipedia.org/wiki/Bipartite_graph)
```

Comment:
Provide it in Markdown format.
```suggestion
 *	@brief Algorithm to check whether a graph is [bipartite](https://en.wikipedia.org/wiki/Bipartite_graph)
```